main.js

```diff
@@ -66,200 +104,134 @@ function displayArticles() {
         keywordsPara.appendChild(keywordSpan);
       });
 
-      searchResults.appendChild(articleDiv);
+      articlesDiv.appendChild(articleContainer);
     });
-  }
-}
-
-function updateNavDisplay() {
-  const nav = document.querySelector('nav');
-  const sortDiv = document.querySelector('.sort-controls');
-
-  if (totalHits > 0) {
+		
     if (resultsPage === 0) {
```

Comment:
what is the difference beteween `==` and `===` again?

Reply:
`===` compares not only the value, but also the data type. So, if the two items being compared are of different data types, strict equals will return `false`. `==` on the other hand tries to convert the values to the same type before comparing, so it may potentially return `true` in cases where strict equals would return `false` (e.g. `'0' == 0`).

Do you think `===` is overkill since we only ever expect `resultsPage` to be an integer?

Reply:
na, I think it is fine the way it is. I just wanted to make sure we understand the implications. thanks!